src/cid.js

```diff
@@ -138,6 +138,14 @@ export class CID {
     }
   }
 
+  /**
+   * @param {string} cidString
+   * @returns {string}
+   */
+  truncate (cidString) {
+    return `${cidString.substr(0, 4)}...${cidString.substr(cidString.length - 4, cidString.length)}`
+  }
```

Comment:
I think we need to decide if this should be either 
(1) a `static` utility function similar to `asCID` or 
(2) something on the instance, like `toString(base)`:

The latter is a bit safer, because we know it is a valid CID, and don't need to worry about edge cases.

### (1)

```suggestion
  static truncate (cidString) {
    return `${cidString.substr(0, 4)}...${cidString.substr(cidString.length - 4, cidString.length)}`
  }
```
### (2) 

```suggestion
  toTruncatedString (base) {
    const cidString =  this.toString(base)
    return `${cidString.substr(0, 4)}...${cidString.substr(cidString.length - 4, cidString.length)}`
  }
```